webui/react/src/components/UserSelectFilter.stories.tsx

```diff
@@ -24,9 +24,9 @@ const UserSelectFilterWithUsers: React.FC<Props> = ({ value }: Props) => {
       type: Users.ActionType.Set,
       value: {
         data: [
-          { id: 1, isActive: true, isAdmin: true, username: 'admin' },
-          { id: 2, isActive: true, isAdmin: false, username: 'user' },
-          { id: 3, isActive: false, isAdmin: false, username: 'inactive' },
+          { isActive: true, isAdmin: true, username: 'admin' },
+          { isActive: true, isAdmin: false, username: 'user' },
+          { isActive: false, isAdmin: false, username: 'inactive' },
```

Comment:
**question:** do we guarantee that `username` is unique?

Reply:
from my previous talks with yoni id was explicitly not exposed because the idea was to rely on user name as the unique identifier. username will be unique however it could potentially change in future if we add that option.